### PR TITLE
[ZEPPELIN-5629] spark.driver.extraJavaOptions setting in %spark.conf does not work well

### PR DESCRIPTION
**Pull request by huage1994**
### What is this PR for?
The value of `spark.driver.extraJavaOptions` setting in `%spark.conf` will override the its initial value provided by zeppelin. It should append instead of override.
This PR fix this bug. 

### What type of PR is it?
[Bug Fix]

### Todos
* [ ] - Task

### What is the Jira issue?
* Open an issue on Jira https://issues.apache.org/jira/browse/ZEPPELIN-5629

### How should this be tested?
* CI passed

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/launcher/SparkInterpreterLauncher.java [hunk 1]
```diff
@@ -77,6 +72,10 @@ public SparkInterpreterLauncher(ZeppelinConfiguration zConf, RecoveryStorage rec
         env.put(key, propValue);
       }
       if (isSparkConf(key, propValue)) {
+        if (Objects.equals("spark.driver.extraJavaOptions", key)) {
```
Comment (zjffdu) [inline zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/launcher/SparkInterpreterLauncher.java:75]: It's better to add a comment explaining why handle `spark.driver.extraJavaOptions` in a specific way?

Comment (huage1994) [reply zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/launcher/SparkInterpreterLauncher.java:80]: thanks a lot for review  @zjffdu,  it's done.

**Comment by zjffdu** [comment]
@huage1994 Could you rebase to the lastest master?

**Comment by huage1994** [comment]
> @huage1994 Could you rebase to the lastest master?

Yeah sure

**Review by zjffdu — APPROVED** [review]

**Pull request closed without merging**